### ISSUE TITLE
Docker image `pext/yotta:gcc5` not available in Docker Hub

### ISSUE THREAD

**Issue by microbit-carlos** [microsoft/pxt-microbit#5313]
So we are unable to fetch this image for local or CI builds:

https://github.com/microsoft/pxt-microbit/blob/58aff6d05af676691536d028dae29256779cecd8/pxtarget.json#L171-L178

- https://hub.docker.com/r/pext/yotta/tags
- https://hub.docker.com/r/pext/yotta/tags?page=1&name=gcc5

**Comment by microbit-carlos** [comment]
@eanders-ms can this image be pushed to DockerHub, or is there any other public registry that could be used in the meantime?

**Comment by JohnVidler** [comment]
As a possible alternative registry - GitHub itself can be used perhaps?

**Comment by microbit-carlos** [comment]
@abchatra @eanders-ms could we get the image pushed to dockerhub or more info on the version of the tools installed?

We are looking into the tooling in general and knowing the version of  thing like `arm-none-eabi-gcc` and `cmake` is important to ensure we don't accidentally make changes that don't work in the older versions from the MakeCode docker image.

**Comment by abchatra** [comment]
@aznhassan ?

**Comment by microbit-carlos** [comment]
Any updates on this?
We had to turn off some of the CODAL CI running MakeCode test builds because `pxt buildtarget` cannot access this docker image (we can use the `PXT_NODOCKER` option, but we want to make sure builds in the docker image don't fail).

**Comment by rafaeliga** [comment]
Im trying to build/run the project on my local mac and got this error...
Its expected? Is there another image that I can use?

**Comment by abchatra** [comment]
@eanders-ms can we push the images to docker?

**Comment by eanders-ms** [comment]
Sorry for the slow response to this! I've pushed the image to docker hub, but the local build is giving me errors:
```
    /usr/bin/arm-none-eabi-gcc

  is not a full path to an existing compiler tool.
```

Troubleshooting it now.

**Comment by microbit-carlos** [comment]
Looking at the image fro Docker hub (`docker run -it --rm pext/yotta:gcc5`), it looks like in this image `arm-none-eabi-gcc` is installed in a different directory.
Does that cause any issues when trying to use MakeCode locally? Or does it just expect the `arm-none-eabi-gcc` executable to be in the PATH?

```
root@e710c3ade4d8:/# /usr/bin/arm-none-eabi-gcc --version
bash: /usr/bin/arm-none-eabi-gcc: No such file or directory
```
```
root@e710c3ade4d8:/# arm-none-eabi-gcc --version
arm-none-eabi-gcc (GNU Tools for ARM Embedded Processors) 5.4.1 20160919 (release) [ARM/embedded-5-branch revision 240496]
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```
```
root@e710c3ade4d8:/# which arm-none-eabi-gcc
/opt/gcc-arm-none-eabi-5_4-2016q3/bin/arm-none-eabi-gcc
```

**Comment by microbit-carlos** [comment]
> Sorry for the slow response to this! I've pushed the image to docker hub, but the local build is giving me errors:
> 
> ```
>     /usr/bin/arm-none-eabi-gcc
> 
>   is not a full path to an existing compiler tool.
> ```
> 
> Troubleshooting it now.

@eanders-ms is this still an issue building for you?

**Comment by abchatra** [comment]
@riknoll  you are addressing this right?

**Comment by riknoll** [comment]
correct, this is one of the images that will be published to the microsoft container registry

**Comment by microbit-carlos** [comment]
@riknoll have the images been migrated to the Microsoft container registry already?
Will the url/identifier of the images listed in the `pxtarget.json` be updated to point to the different container registry?

**Comment by abchatra** [comment]
If this is still an issue reopen.

**Comment by microbit-carlos** [comment]
The `pext/yotta:gcc5` image is available in DockerHub, so that's good. The only thing I was wondering is if these images were going to be ported to the Microsoft container registry, and if that's the case, if the pxtarget.json file would be updated to point to the different registry.
